quantlib/test/test_termstructures.py

```diff
@@ -120,3 +125,57 @@ def test_reference_evaluation_data_changed(self):
         for i, val in enumerate(expected):
             self.assertAlmostEquals(val, calculated[i])
 
+class ForwardSpreadedTestCase(unittest.TestCase): 
+    
+    def test_forward_spreaded_ts(self):
+        m = libor_market('USD(NY)')
+        eval_date = Date(20, 9, 2004)
+        
+        quotes = [('DEP', '1W', SimpleQuote(0.0382)),
+                    ('DEP', '1M', SimpleQuote(0.0372)),
+                    ('DEP', '3M', SimpleQuote(0.0363)),
+                    ('DEP', '6M', SimpleQuote(0.0353)),
+                    ('DEP', '9M', SimpleQuote(0.0348)),
+                    ('DEP', '1Y', SimpleQuote(0.0345))]
+        
+        m.set_quotes(eval_date, quotes)
+        ts = m.bootstrap_term_structure()
+        
+        # We don't test for numerical accuracy.
+        #self.assertGreater(zero_rate.rate, 0)
+        #self.assertGreater(forward_rate.rate, 0)
+        
+        # Check that the linked term structures are consistent with the
+        # original term structure.
+        discount_ts = m._discount_term_structure
+        forecast_ts = m._forecast_term_structure
+        discount_spd = 0.05
+        forecast_spd = 0.08
+        
+        fwd_spd_dts = ForwardSpreadedTermStructure(discount_ts,SimpleQuote(discount_spd))
```

Comment:
we try to enforce pep8 compliance in the code. This would need a space after the comma (see https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements). Same comments for the other lines below.